### PR TITLE
Symlinks should be preferred on OSes that support them

### PR DESCRIPTION
**Pull request by cryos**
This leads to less confusion when editing the Python files, and clearly
expresses our intent when the operating system support is available.

Add the Python files to our SOURCES list so that they show up in IDEs.

### CODE REVIEW

File: tomviz/CMakeLists.txt [hunk 1]
```diff
@@ -158,10 +158,22 @@ set(python_files
...
+make_directory("${tomviz_BINARY_DIR}/share/tomviz/scripts")
 foreach(file ${python_files})
-  set(file python/${file})
-  file(COPY ${file} DESTINATION "${tomviz_BINARY_DIR}/share/tomviz/scripts")
-  install(FILES ${file} DESTINATION "${tomviz_data_install_dir}/scripts" COMPONENT "Scripts")
+  list(APPEND SOURCES "python/${file}")
+  execute_process(COMMAND ${CMAKE_COMMAND} -E ${script_cmd}
+    "${tomviz_SOURCE_DIR}/tomviz/python/${file}"
+    "${tomviz_BINARY_DIR}/share/tomviz/scripts/${file}")
+  install(FILES ${file}
```
Comment (mathturtle) [inline tomviz/CMakeLists.txt]: This should be `install(FILES python/${file}`.  Right now `make install` fails since it cannot find the file.

Comment (cryos) [inline tomviz/CMakeLists.txt]: Yes, I forgot to test make install. Good catch - thanks Shawn!

**Comment by cryos** [comment]
Should work now

**Comment by mathturtle** [comment]
`make install` works now.  Hopefully this is all that is required to avoid breaking superbuild.

**Comment by cryos** [comment]
It should be, this topic is much more about the build tree when actively developing, and far less about the install tree.

**Pull request merged**